client/js/_author/components/assessments/_edit_assessment.jsx

```diff
@@ -81,29 +75,13 @@ export class EditAssessment extends React.Component {
     );
   }
 
-  updateAssessment() {
+  updateAssessment(newFields) {
     this.props.updateAssessment(
       this.props.params.bankId,
-      {
-        name: this.state.assessment.name,
-        description: this.state.assessment.description,
-        id: this.props.params.id,
-      },
+      { ...{ id: this.props.params.id }, ...newFields },
```

Comment:
why bother with ...{ id: this.props.params.id }, why not just `{ id: this.props.params.id, ...newFields }`